api/src/lib/interface.ts

```diff
@@ -571,6 +571,12 @@ export interface DataCiteCreator {
     nameIdentifiers: DataCiteCreatorNameIdentifiers[];
 }
 
+export interface DataCiteDoiReferences {
```

Comment:
This can be removed, it's not being used anywhere.

Reply:
Good spot - I've deleted it ☺️